utils/collectd/Makefile

```diff
@@ -146,6 +146,7 @@ COLLECTD_PLUGINS_SELECTED:= \
 	df \
 	disk \
 	dns \
+	dhcpleases \
```

Comment:
This line is not sorted alphabetically.   
df < dhcpleases < disk < dns

Same goes also for the eval BuildPlugin list below list .

And seems to be a problem also in the upstream patch. You have generally placed the dhcp... line/section as the "last of d" instead of sorted properly into middle of the d items.

Reply:
changed everywhere, even upstream plugin. ;)